src/raster/r.lake.series/r.lake.series.py

```diff
@@ -132,9 +132,10 @@ def format_order(number, zeros):
 
 
 def frange(x, y, step):
-    while x <= y:
-        yield x
-        x += step
+    return [
+        val / 1000
```

Comment:
Suggestion:
```
exponent  =  abs(decimal.Decimal(step_as_string).as_tuple().exponent)
```
and use `10 ** exponent` instead of fixed 1000 and then use that for formatting (e.g. `f"{prefix}_{water_level:.{exponent}f}"`)

Reply:
Nice work around. I implemented it in the latest commit. Thanks!

Reply:
I made the changes we talked about and renamed exponent to precision since I think that's a more intuitive name.